provider/datagen/src/bin/datagen/args.rs

```diff
@@ -233,6 +241,9 @@ pub struct Cli {
 
     #[arg(short, long, value_enum, default_value_t = Fallback::Legacy)]
     fallback: Fallback,
+
+    #[arg(long, value_enum, default_value_t = RegionInclude::DefaultOnly)]
```

Comment:
Issue: This should default to All / AllInherited. DefaultOnly only makes sense when there was an explicit list of locales.

Reply:
Current behaviour is `ExplicitOnly`, so this would be quite the behaviour change.